### PR TITLE
Add config map to example Addon Resizer deployment

### PR DESCRIPTION
**Pull request by kawych**

### CODE REVIEW

File: addon-resizer/README.md [hunk 1]
```diff
@@ -35,6 +35,19 @@ Usage of pod_nanny:
 The following yaml is an example deployment where the nanny watches and resizes itself.
 
 ```yaml
+# Config map for resource configuration.
+# Specify 'cpu', 'extra-cpu', 'memory' and 'extra-memory'
+# to overwrite resource requirements.
+apiVersion: v1
```
Comment (losipiuk) [inline addon-resizer/README.md]: Was the example broken without this section?

Comment (kawych) [reply addon-resizer/README.md]: No, but I want to include a deployment with config map to give an example how to use it.

Comment (losipiuk) [reply addon-resizer/README.md]: Cool. Maybe it is worthy to note the intention in description above the yaml snippet?

**Pull request merged**